src/components/ReportModal/ReportModal.tsx

```diff
@@ -21,12 +24,12 @@ const ReportModal: FC<ReportModalInterface> = ({ onConfirm, onDecline }) => {
   ];
 
   const observationField = [
-    { name: 'Primary Cancer', value: undefined },
-    { name: 'Laterality', value: undefined },
-    { name: 'Clinical TNM', value: undefined },
-    { name: 'Estrogen Receptor', value: undefined },
-    { name: 'Progesterone Receptor', value: undefined },
-    { name: 'HER2 Receptor', value: undefined }
+    { name: 'Primary Cancer', value: mcodeRecords.primaryCancer ?? 'Unknown' },
+    { name: 'Laterality', value: mcodeRecords.laterality ?? 'Unknown' },
+    { name: 'Clinical TNM', value: getTNM(mcodeRecords) },
```

Comment:
should this one also have `?? 'Unknown'` ?

Reply:
No, `getTNM` always returns a string. However if all three TNM values are undefined the result will be "   " so I will add a check to the method so it returns "Unknown" if all 3 are undefined